src/Microsoft.Diagnostics.EventFlow.Outputs.ElasticSearch/ElasticSearchOutput.cs

```diff
@@ -98,7 +98,7 @@ public async Task SendEventsAsync(IReadOnlyCollection<EventData> events, long tr
                 // Note: the NEST client is documented to be thread-safe so it should be OK to just reuse the this.esClient instance
                 // between different SendEventsAsync callbacks.
                 // Reference: https://www.elastic.co/blog/nest-and-elasticsearch-net-1-3
-                IBulkResponse response = await this.connectionData.Client.BulkAsync(request).ConfigureAwait(false);
+                BulkResponse response = await this.connectionData.Client.BulkAsync(request).ConfigureAwait(false);
```

Comment:
nit: Any reason to use class instead of an interface for the response?

Reply:
Good question--the reason is Elasticsearch decided to change the interface and use classes instead; `IBulkResponse` is no more ☺️